roles/monitoring_sap/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Do a general check for platform architecture

Reply:
the platform check and the release-version ch3eck is separated only because of the different error message,
in case one check fails.